spacy/tests/parser/test_add_label.py

```diff
@@ -123,6 +123,7 @@ def test_ner_labels_added_implicitly_on_predict():
     assert "D" in ner.labels
 
 
+@pytest.mark.skip(reason="Not yet supported")
```

Comment:
We probably want to use `xfail` than `skip` for this test (with the reason set `"no beam parser yet"`), I think.